cg/server/ext.py

```diff
@@ -123,6 +126,7 @@ def init_app(self, app):
 microsalt_validation_service = MicroSaltValidationService(store=db)
 mip_dna_validation_service = MipDnaValidationService(store=db)
 mutant_validation_service = MutantValidationService(store=db)
+pacbio_long_read_validation_service = PacbioLongReadValidationService(store=db)
```

Comment:
We should move these into a registry

Reply:
Yes, we definitely should tidy up how these are instantiated. My hunch is that we might not want these services order type specific in the long run even. If they accepted the lists of rules and model (e.g. TomteOrder) as input in the .validation method we could have one service fits all order types.

Reply:
But I would probably consider that a separate issue from the Pacbio service addition

Reply:
> Yes, we definitely should tidy up how these are instantiated. My hunch is that we might not want these services order type specific in the long run even. If they accepted the lists of rules and model (e.g. TomteOrder) as input in the .validation method we could have one service fits all order types.

You mean have one "validator" in which you inject a ruleset?

Reply:
Yeah pretty much! I guess we would have some sort of registry where you would provide the order type and get a ruleset and a Basemodel to validate towards? And then inject them into a common "validator"

Reply:
closing as out of scope. There is a lot to polish but this will be done in the dev branch